docs/process.md

```diff
@@ -84,12 +91,16 @@ What should reviewers be checking for?
 
 ### Stage 4: Social media
 
-Once published, the writer creates social media posts and shares links on the “DCR Social Media” channel on Matrix. This gives the wider community an opportunity to engage with the article, adding comments, likes and retweets etc.
+Once published, the writer creates social media posts and shares links on the “DCR Social Media” and "DCR Writers" channels on Matrix. This gives the wider community an opportunity to engage with the article, adding comments, likes and retweets etc.
 
 What social media content should writers create?
 
-* Twitter — Initial tweet from writers' Twitter account, maybe do this as a thread and put the link on the last post. They could also schedule a tweet a few days later, perhaps with a poll attached to increase engagement.
+* Twitter — Initial tweet from writers' Twitter account, this could be done as a single post or thread. Don't forget to include the link.
```

Comment:
Maybe clarify how does this play together with "official" release tweet from the `cypherpunktimes` account. Should authors *always* tweet from their individual accounts, or is it a recommendation? Should they tweet it before or after the `cypherpunktimes` tweet?

Reply:
At the moment, the individual author accounts get more exposure, so I wonder if it’s best to post from individual accounts and then retweet from the CT account. What do you think?

Reply:
I would suggest to grow the `@cypherpunktimes` account as the first priority. Content is getting better but the audience and the Twitter account are still very small. We need a smart strategy to increase its reach on (Crypto) Twitter and get more readers (and writers).

It's hard to say which procedure is the best since I'm not a Twitter algo expert. But I suspect announcing a new article from CT account first is better, it feels like "more original" content compared to retweeting.

After some time you can retweeting a tweet from author's individual account too, that doesn't hurt.

And maybe check what big crypto outlets are doing with their Twitter, like do they prefer own tweets or retweet.s

Reply:
Done 👍